lib/android/audio.nit

```diff
@@ -537,8 +536,8 @@ redef class Sound
 	end
 
 	redef fun play do
-		if self.error != null then return
 		if not is_loaded then load
+		if self.error != null then return
```

Comment:
Why moving these 2 lines? it will try to load the file at each usage even if it doesn't exist...